app/src/test/java/org/feature/fox/coffee_counter/di/services/network/ApiServiceTest.kt

```diff
@@ -365,7 +367,7 @@ class ApiServiceTest {
 
         mockWebServer.enqueue(response)
 
-        val actualResponse = apiService.getTransactions(userId)
+        val actualResponse = apiService.getTransactions(bearerToken, userId)
```

Comment:
The test cases should also assert if the authorization header was send correctly to mock webserver